frontend/app/src/utils/log-level.ts

```diff
@@ -1,10 +1,20 @@
+export const TRACE = 'trace';
 export const DEBUG = 'debug';
 export const INFO = 'info';
 export const WARN = 'warn';
 export const ERROR = 'error';
+export const SILENT = 'silent';
```

Comment:
I don't think we need silent/trace here. 
The highest we can go is critical and we are not really using trace, are we?

Reply:
This is to keep the functionality in logging.ts don't break. I put all possible levels in `loglevels` package here.
```
    if (levels.indexOf(methodName as Level) >= logLevel) {
      if (interop.isPackaged) {
        interop.logToFile(`(${methodName}): ${message.join('')}`);
      }
    }
```

Reply:
Then we can at least avoid exporting them since we use them only internally

Reply:
Since we are only using them internally, would it be possible to remove the export?